src/fparser/two/utils.py

```diff
@@ -0,0 +1,1216 @@
+#!/usr/bin/env python
```

Comment:
I think we only need this line if the file is to be executable as it tells bash how to interpret the script.

Reply:
Removed.